datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/common/mappers/CostValueMapper.java

```diff
@@ -16,7 +16,7 @@ public static CostValue map(@NonNull final com.linkedin.common.CostValue costVal
     public CostValue apply(@NonNull final com.linkedin.common.CostValue costValue) {
         final CostValue result = new CostValue();
         result.setCostCode(costValue.getCostCode());
-        result.setCostId(costValue.getCostId());
+        result.setCostId(costValue.getCostId().floatValue());
```

Comment:
Is this an intended change? @jjoyce0510

Reply:
Yes - These are represented as floats in the GQL model